src/views/CountryProfile.js

```diff
@@ -74,24 +87,59 @@ const mapOptions = {
     }]
   }
 
-
+  function loadCountryData(countryId){
+    //alert(countryId);
+    var countryData = require('../assets/data/countryProfile.json');
```

Comment:
Consider using let. You can read more [here](https://dev.to/sarah_chima/var-let-and-const--whats-the-difference-69e)